.github/workflows/_binary-upload.yml

```diff
@@ -104,6 +103,13 @@ jobs:
         with:
           no-sudo: true
 
+      - name: Configure aws credentials (pytorch account)
+        if: ${{ github.event_name == 'push' && (github.event.ref == 'refs/heads/nightly' || (startsWith(github.event.ref, 'refs/tags/') && !startsWith(github.event.ref, 'refs/tags/ciflow/'))) }}
```

Comment:
Do you think we will need to split up the if condition here for nightly and test because they are under two different rules `gha_workflow_nightly_build_wheels` and `gha_workflow_test_build_wheels`.  They are kind of similar for now, but that might not be the case going forward.